docs/examples/nginx/README.md

```diff
@@ -0,0 +1,29 @@
+# nginx: An advanced example chart
+
+This Helm chart provides examples of some of Helm's more powerful
+features.
+
+**This is not a production-grade chart. It is an example.**
+
+The chart installs a simple nginx server according to the following
+pattern:
+
+- A `ConfigMap` is used to store the files the server will serve.
+  (`templates/configmap.yaml`)
+- A `Deployment` is used to create a Replica Set of nginx pods.
+  (`templates/deployment.yaml`)
+- A `Service` is used to create a gateway to the pods running in the
+  replica set (`templates/svc.yaml`)
+
+The `values.yaml` exposes a few of the configuration options in the
+charts, though there are some that are not exposed there (like
+`.image`).
+
+The `templates/_helpers.tpl` file contains helper templates. The leading
```

Comment:
This makes me wonder if user create some number of such files, and if so why would the need to?